pkg/drivers/http/options.go

```diff
@@ -26,6 +29,7 @@ func newOptions(setters []Option) *Options {
 	opts.Backoff = pester.ExponentialBackoff
 	opts.Concurrency = 3
 	opts.MaxRetries = 5
+	opts.AllowedHTTPCodes = []int{stdhttp.StatusOK}
```

Comment:
Maybe ``map[int]struct{}`` instead of the slice?

You could change the existence check later like this:
```go
_, exists := drv.options.AllowedHTTPCodes[resp.StatusCode]

return true
```

Reply:
Sure, I changed it. 

Why do you think this way is better? I think that using a map is even a bit slower because the map must be initialised and more time will be used on the hash table lookup of int than just iterating through a slice with < 20 elements in it.